### DISCUSSION TITLE
静态资源换源

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#2219] [Ideas]
建议所有存放在 github 上的静态资源都更换成这个源：https://github.moeyy.xyz/

例：https://github.moeyy.xyz/https://raw.githubusercontent.com/banned-historical-archives/banned-historical-archives7/main/7d/7dd3ce92-638a-4315-a46d-4408ce61e295.png

@banned-historical-archives github 的大陆访问情况堪忧，这个源的网络表现非常优秀

**Comment by banned-historical-archives** [comment]
这可能不是好注意
1） 稳定性未知，网站随时可能关闭
2）访问的链接可能和真实内容不一致，可能被拦截/替换/篡改
3）可能的其他风险。这个网站本来应该翻墙使用，否则可能被追踪

**Reply by PirateBook1** [reply]
有道理，我轻视安全性了
